src/Uno.UI/UI/Xaml/IFrameworkElement.cs

```diff
@@ -252,9 +252,26 @@ public static IFrameworkElement FindName(IFrameworkElement e, IEnumerable<View>
 				}
 			}
 
+			if(e is UIElement uiElement && uiElement.ContextFlyout is Controls.Primitives.FlyoutBase contextFlyout)
+			{
+				return FindInFlyout(name, contextFlyout);
+			}
+
+			if (e is Button button && button.Flyout is Controls.Primitives.FlyoutBase buttonFlyout)
+			{
+				return FindInFlyout(name, buttonFlyout);
+			}
+
```

Comment:
Even if not yet supported by Uno, `SplitButton` also have a `Flyout` property. I think that we are able to support it immediately, otherwise  we should find a way to not forget it when we will implement the `SplitButton`.

Reply:
There's no common technique to validate this behavior but we'll have to align onto what WinUI does.